source/FreeRTOS_DNS_Cache.c

```diff
@@ -173,9 +173,9 @@
         { /* Element found */
             if( xLookUp == pdTRUE )
             {
-                prvGetCacheIPEntry( uxIndex,
-                                    pulIP,
-                                    ulCurrentTimeSeconds );
+                ( void ) prvGetCacheIPEntry( uxIndex,
```

Comment:
Maybe we should add a comment about why the result is being discarded here and why is it safe to do so.
Something along the lines of:
```
/* This statement can only be reached when xResult is true; which
 * implies that the entry is present and a 'get' operation will result
 * in success. Therefore, it is safe to ignore the return value of the
 * below function. */
```

Reply:
We have too many (void) in front of functions (e.g. thousands of them including memcpy) .  I don't think we need to add for one specific function calls.